packages/@react-aria/dnd/src/useDraggableItem.ts

```diff
@@ -65,7 +67,7 @@ const MESSAGES = {
  */
 export function useDraggableItem(props: DraggableItemProps, state: DraggableCollectionState): DraggableItemResult {
   let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-aria/dnd');
-  let isDisabled = state.selectionManager.isDisabled(props.key);
+  let isDisabled = props.isDisabled || state.selectionManager.isDisabled(props.key);
```

Comment:
Discussed with the team some more about this, and it might be better if we disabled all the draggable items via the state. The reason for this is because by adding this prop someone could disable any arbitrary draggable item, resulting in cases where a user might select multiple items (including the disabled draggable items) -> drag the items -> wonder why some items aren't being dragged. Disabling dragging still makes sense in the case where ALL items should have their draggability disabled though

Reply:
Thank you for your comment. You're right, there might indeed be a chance of users accidentally misoperating. I'll try to see if it can be fixed.